products/cakephp.md

```diff
@@ -140,6 +140,85 @@ releases:
     releaseDate: 2017-07-22
     latestReleaseDate: 2020-12-15
     link: https://bakery.cakephp.org/2020/12/15/cakephp_21024_released.html
+    
+-   releaseCycle: "2.9"
+    latest: "2.9.9"
+    eol: true
+    support: false
+    releaseDate: 2016-09-19
+    latestReleaseDate: 2017-05-26
+    link: https://bakery.cakephp.org/2017/05/25/cakephp_299_released.html    
+    
+-   releaseCycle: "2.8"
+    latest: "2.8.9"
+    eol: true
+    support: false
+    releaseDate: 2016-02-07
+    latestReleaseDate: 2016-09-19
+    link: https://bakery.cakephp.org/2016/09/18/cakephp_290_289_released.html
+    
+-   releaseCycle: "2.7"
+    latest: "2.7.11"
+    eol: true
+    support: false
+    releaseDate: 2015-07-12
+    latestReleaseDate: 2016-03-14
+    link: https://bakery.cakephp.org/2016/03/13/cakephp_2613_2711_282_3017_3112_325_released.html
+ 
+-   releaseCycle: "2.6"
+    latest: "2.6.7"
+    eol: true
+    support: false
+    releaseDate: 2014-12-23
+    latestReleaseDate: 2015-05-29
+    link: https://bakery.cakephp.org/2015/05/28/cakephp_2_6_7_released.html
```

Comment:
```suggestion
    latest: "2.6.13"
    eol: true
    support: false
    releaseDate: 2014-12-23
    latestReleaseDate: 2016-03-14 
    link: https://bakery.cakephp.org/2016/03/13/cakephp_2613_2711_282_3017_3112_325_released.html
```

Reply:
@ravage84 lets make this one as like @captn3m0 suggested in #1708

Reply:
i will be glad if you review that one too